evaluation_scripts/Score_Weekly.py

```diff
@@ -140,67 +140,27 @@
 
 
 # %%
+# Week 9 addition:
+# Replaced plotting script with functions
 
 # Add Histogram of results, plots each student's guess,
-# and the actual mean value for week one.
-plt.figure(figsize=(8, 6))
-plt.hist(forecasts1, bins=120, color='blue', alpha=0.75,
-         label='Student Guesses')
-plt.plot([obs_week]*3, np.arange(0, 3, 1), color='red',
-         linestyle='-', label='Actual mean')
-plt.title('Student Guesses, week 1')
-plt.xlabel('Flow Forecast')
-plt.ylabel('Count')
-plt.legend(loc='upper left')
-# %%
+# and the actual mean value for week 1 and week 2.
+title_string1 = 'Student guesses for Week 1'
+histogram1 = pf.get_histogram(forecasts1, obs_week, title_string1)
```

Comment:
I like how you did this. You could simplify it a bit  by just having the forecast week # (i.e.  1 or 2) as an input to your function rather than 'title string1' then the title_string can be created within your function and you can make the plot with just one line call rather than 2.

Reply:
Thanks for your comment. The function was modified correspondingly in plot_functions.py.